draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1094,28 +1094,28 @@
    caches can use them to keep their contents up to date.
 </t>
 <t>
-   A cache &MUST; invalidate the effective Request URI
-   (<xref target="effective.request.uri"/>) as well as the URI(s) in the
+   A cache &MUST; invalidate the target URI
+   (<xref target="target.resource"/>) as well as the URI(s) in the
    <x:ref>Location</x:ref> and <x:ref>Content-Location</x:ref> response header
    fields (if present) when a non-error status code is received in response to
    an unsafe request method.
 </t>
 <t>
    However, a cache &MUST-NOT; invalidate a URI from a <x:ref>Location</x:ref>
    or <x:ref>Content-Location</x:ref> response header field if the host part of
-   that URI differs from the host part in the effective request URI
-   (<xref target="effective.request.uri"/>). This helps prevent denial-of-service attacks.
+   that URI differs from the host part in the target URI
+   (<xref target="target.resource"/>). This helps prevent denial-of-service attacks.
 </t>
 <t>
-   A cache &MUST; invalidate the effective request URI 
-   (<xref target="effective.request.uri"/>) when it receives a non-error response 
+   A cache &MUST; invalidate the target URI 
+   (<xref target="target.resource"/>) when it receives a non-error response 
    to a request with a method whose safety is unknown.
 </t>
 <t>
    Here, a "non-error response" is one with a <x:ref>2xx (Successful)</x:ref>
    or <x:ref>3xx (Redirection)</x:ref> status code. "Invalidate" means that
```

Comment:
Oh, so this is a defined term? That's weird. It should at least be defined before it is used, but would prefer to just inline the phrase above.

Reply:
Reading it, I agree -- will address in a new issue.